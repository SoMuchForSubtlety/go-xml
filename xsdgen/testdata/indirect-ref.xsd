<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ns="http://example.org/"
  targetNamespace="http://example.org/">
  <xs:complexType name="Type1">
    <xs:sequence>
      <xs:element maxOccurs="1" minOccurs="0" ref="ns:ReferencedType">
      </xs:element>
    </xs:sequence>
  </xs:complexType>
  <xs:complexType name="ReferencedType">
    <xs:sequence>
      <xs:element maxOccurs="1" minOccurs="1" name="fild1" nillable="false" type="xs:string">
      </xs:element>
    </xs:sequence>
  </xs:complexType>
  <xs:element name="ReferencedType" type="ns:ReferencedType" />
</xs:schema>