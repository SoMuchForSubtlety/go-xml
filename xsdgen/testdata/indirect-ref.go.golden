// Code generated by xsdgen.test. DO NOT EDIT.

package ws

type ReferencedType struct {
	Field1 string `xml:"http://example.org/ field1"`
}

type Type1 struct {
	ReferencedType *ReferencedType `xml:"http://example.org/ ReferencedType,omitempty"`
}
